src/containers/InteractiveMessage/InteractiveOptions/QuickReplyTemplate.tsx

```diff
@@ -50,13 +50,18 @@ export const QuickReplyTemplate: React.SFC<QuickReplyTemplateProps> = (props) =>
             onChange={(e: any) => handleInputChange(e, 'value')}
             value={defaultValue}
             className={styles.TextField}
+            InputProps={{
+              endAdornment:
+                inputFields.length > 1 ? (
+                  <CrossIcon className={styles.RemoveIcon} title="Remove" onClick={onRemoveClick} />
+                ) : null,
```

Comment:
```suggestion
                inputFields.length > 1 && (
                  <CrossIcon className={styles.RemoveIcon} title="Remove" onClick={onRemoveClick} />
                ) ,
```

Let's assign this and adornment as a const above and use it here. WIll be better for code readability.